README.md

```diff
@@ -308,6 +308,12 @@ The most recent version of the project Docker image, which is pushed to Docker H
 docker pull pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:latest
 ```
 
+Development using the project Docker image is highly recommended. An analysis that is developed using the project Docker image can be efficiently rerun by another developer or the original developer (after a long time since it is developed), without dependency or numerical issues. This would significantly facilitate the following tasks that are constantly performed by all developers of the OpenPedCan-analysis project.
```

Comment:
```suggestion
Development should utilize the project Docker image. 
An analysis that is developed using the project Docker image can be efficiently rerun by another developer or the original developer (after a long time since it is developed), without dependency or numerical issues. 
This will significantly facilitate the following tasks that are constantly performed by all developers of the OpenPedCan-analysis project.
```

Reply:
Should we remove the "Local Development" section, as we require using Docker image to develop?

Reply:
we can perhaps leave it for some edge cases - and we may need to add some language later about development of modules for CAVATICA, but I don't think we have the rules yet.

Reply:
> we can perhaps leave it for some edge cases - and we may need to add some language later about development of modules for CAVATICA, but I don't think we have the rules yet.

Sounds good. I will leave the "Local Development" section there.